qcodes/dataset/sqlite/__init__.py

```diff
@@ -0,0 +1,22 @@
+"""
+This module encapsulates QCoDeS database: its schema, structure, convenient
+and relevant queries, wrapping around :mod:`sqlite3`, etc.
+
+The dependency structure of the sub-modules is the following:
+
+::
+
+         .connection     .settings
+           /   |   \         |
+          /    |    \        |
+         /     |     V       V
+        |      |  .query_helpers
+        |      |   |       |
+        |      V   V       |
+        |  .db_upgrades    |
+        |      /           V
+        |     /        .queries
+        v    v
+      .database
+
```

Comment:
Cool, now we just need to remember to keep this up to date

Reply:
yes ;) that's why the text above the diagram a bit vague :)